interface/usergroup/user_admin.php

```diff
@@ -425,6 +429,7 @@ class="form-control" value="<?php echo attr($frow['id']); ?>"><?php echo text($f
 </tr>
 <tr>
 <td><span class="text"><?php echo xlt('Weno Provider ID'); ?>: </span></td><td><input type="text" name="erxprid" style="width:150px;" class="form-control" value="<?php echo attr($iter["weno_prov_id"]); ?>"></td>
+<td><span class="text"><?php echo xlt('Google Email for Login'); ?>: </span></td><td><input type="text" name="google_signin_email" style="width:150px;" class="form-control" value="<?php echo attr($iter["google_signin_email"]); ?>"></td>
```

Comment:
just realized, this will also need to be added to the interface/usergroup/usergroup_admin_add.php script